### PR TITLE
CIV-6104 - clean up jenkins file and charts

### PR DESCRIPTION
**Pull request by chris-pateman**
### JIRA link (if applicable) ###
CIV-6104

### Change description ###
Change keyvaults to disable key vault
remove redundent calls in jenkins file
remove redundent statements in values

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -56,10 +56,6 @@ def publishRpaContract() {
 }
 
 withPipeline(type, product, component) {
-  pipelineConf = config
-  disableLegacyDeployment()
-  enableAksStagingDeployment()
-  enableSlackNotifications('#civil_notifications')
```
Comment (filippo-possenti-version1) [inline Jenkinsfile_CNP:62]: I'm a bit unclear on what you're doing in general in this PR, but this stood out to me.

Is this removal intentional?

**Review by filippo-possenti-version1 — APPROVED** [review]

File: Jenkinsfile_CNP (hunk 1 above)
Comment (filippo-possenti-version1) [reply Jenkinsfile_CNP:62]: (and I mean specifically the one on `enableSlackNotifications`

Comment (chris-pateman) [reply Jenkinsfile_CNP:62]: yes it should be all as defualt so doesn't need calling out.

**Review by asthamalviya — APPROVED** [review]

**Review by sabahirfan — APPROVED** [review]
